tests/005-test-devmapper-cleanup.sh

```diff
@@ -31,13 +31,19 @@ EOF
   $DSSBIN --reset >> $LOGS 2>&1
   # Test failed.
   if [ $? -eq 0 ]; then
-     if [ ! -e /etc/sysconfig/docker-storage ]; then
-          test_status=0
+     if [ -e /etc/sysconfig/docker-storage ]; then
+	 echo "ERROR: $testname: $DSSBIN failed. /etc/sysconfig/docker-storage still exists." >> $LOGS
      else
-	  echo "ERROR: $testname: $DSSBIN failed. /etc/sysconfig/docker-storage still exists." >> $LOGS
+	 if lvm_exists $vg_name "docker-pool"; then
```

Comment:
Looks like bash does not exit if command not found. I see following on stdout and test still passes.

/tests/005-test-devmapper-cleanup.sh: line 38: lvm_exists: command not found